config.yaml.defaults

```diff
@@ -75,17 +75,67 @@ app:
       icon: Info
       url: /about
 
-  homepage_widgets:
-    # Layouts arrays should be [x, y, w, h]
-    # See https://github.com/STRML/react-grid-layout
-
-    # component_props are properties passed down to the
-    # underlying React component, indicated by component_name
+  # See https://github.com/STRML/react-grid-layout for more on the grid package
+  homepage_grid:
+    # This section describes the grid on which Home Page widgets are laid out.
+    #
+    # The breakpoints describe screen sizes at which a different set of widget
+    # layouts should be used.
+    #
+    # The cols describe the number of evenly spaced columns that make up the
+    # grid at a given breakpoint. For example, on extra-large screens (greater
+    # than ${breakpoints.xlg} pixels), the grid will use ${cols.xlg} columns of
+    # equal width to describe sizes of widgets based on the layouts provided.
+    #
+    # Optionally, you may provide a "row_height: {number of pixels}" property in
+    # this section to change the height of a row on the grid. By default, this
+    # value is 150px.
+
+    breakpoints:
+      xlg: 1400
+      lg: 1150
+      md: 996
+      sm: 650
+      xs: 0
```

Comment:
Is it necessary to specify the 0 point (xs) ?

Reply:
I guess it's confusing: anything >0 should fit within the next higher up bracket?

Reply:
Ah yeah, this is confusing but I noticed the breakpoints passed to react-grid-layout aren't quite the same as breakpoints for, say, MUI. These are min-width values instead of max-width values. So the small breakpoint would apply to any window (or rather, content divs) from 650 up to 996, the xs breakpoint would match with windows 0 up to 650, etc. Not sure if I'm missing something, but that's what I see when I hook up to the onBreakpointChange and print out the breakpoints it's using.

Reply:
OK, may be worth pointing that out, since it works differently in MUI where we specify the lower boundaries for transitions.